__tests__/steps/measureStep.spec.ts

```diff
@@ -138,5 +139,14 @@ describe('measureStep', () => {
         },
       });
     });
+    it('should return when the start mark doesnt exist', () => { 
```

Comment:
Empty space missing

Reply:
Also, what should return? you are testing a method to be called, not a value to be returned.

What are we actually testing here?

Reply:
perhaps the naming could be a bit more specific like `should not measure the step when the start mark doesn't exist`